classes/class-wc-connect-compatibility.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * A class for working around the quircks and different versions of WordPress/WooCommerce
+ */
+
+// No direct access please
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! class_exists( 'WC_Connect_Compatibility' ) ) {
+
+	class WC_Connect_Compatibility {
+		private static $singleton;
+
+		public static function instance() {
+			if( self::$singleton === null ) {
+				self::$singleton = new WC_Connect_Compatibility();
+			}
+
+			return self::$singleton;
+		}
+
+		public function __construct() {
+		}
+
+		public function get_order_id( WC_Order $order ) {
```

Comment:
I wonder if we should add a comment about the changes from 2.6->3.0 that require this? Or is that overkill ;)?